docs/tutorials/word2vec.ipynb

```diff
@@ -37,20 +37,20 @@
         "id": "AOpGoE2T-YXS"
```

Comment:
<p>Same here, there appears to be an extra <code>&gt;</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/text/pull/1229/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>